governance/votes/Executive vote - November 26, 2021.md

```diff
@@ -0,0 +1,150 @@
+
+
+
+**DAI Streams**
+
+A total of **1,860,000 DAI** will be streamed to the Real World Finance ([RWF-001](https://mips.makerdao.com/mips/details/MIP39c2SP1)) Core Unit at [0x9e1585d9CA64243CE43D42f7dD7333190F66Ca09](https://etherscan.io/address/0x9e1585d9CA64243CE43D42f7dD7333190F66Ca09) over 12 months starting 2022-01-01 as per their most recent [budget proposal](https://mips.makerdao.com/mips/details/MIP40c3SP39).  
+
+A total of **12,242 DAI** will be streamed to the Governance Communications ([COM-001](https://mips.makerdao.com/mips/details/MIP39c2SP8)) Core Unit at [0x1eE3ECa7aEF17D1e74eD7C447CcBA61aC76aDbA9](https://etherscan.io/address/0x1eE3ECa7aEF17D1e74eD7C447CcBA61aC76aDbA9) over 1 month starting 2021-12-01 as per their most recent [budget proposal](https://mips.makerdao.com/mips/details/MIP40c3SP40).  
+
+A total of **257,500 DAI** will be streamed to the Governance Communications ([COM-001](https://mips.makerdao.com/mips/details/MIP39c2SP8)) Core Unit at [0x1eE3ECa7aEF17D1e74eD7C447CcBA61aC76aDbA9](https://etherscan.io/address/0x1eE3ECa7aEF17D1e74eD7C447CcBA61aC76aDbA9) over 6 months starting 2021-01-01 as per their most recent [budget proposal](https://mips.makerdao.com/mips/details/MIP40c3SP40).  
+
+A total of **1,130,393 DAI** will be streamed to the Sidestream Auction Services ([SAS-001](https://mips.makerdao.com/mips/details/MIP39c2SP25)) Core Unit at [0xb1f950a51516a697E103aaa69E152d839182f6Fe](https://etherscan.io/address/0xb1f950a51516a697E103aaa69E152d839182f6Fe) over 12 months starting 2021-12-01 as per their most recent [budget proposal](https://mips.makerdao.com/mips/details/MIP40c3SP43).  
+
+A total of **366,563 DAI** will be streamed to the Immunefi Security ([IS-001](https://mips.makerdao.com/mips/details/MIP39c2SP24)) Core Unit at [0xd1F2eEf8576736C1EbA36920B957cd2aF07280F4](https://etherscan.io/address/0xd1F2eEf8576736C1EbA36920B957cd2aF07280F4) over 8 months starting 2021-12-01 as per their most recent [budget proposal](https://mips.makerdao.com/mips/details/MIP40c3SP42).  
+
+A total of **424,944 DAI** will be streamed to the Content Production ([MKT-001](https://mips.makerdao.com/mips/details/MIP39c2SP5)) Core Unit at [0xDCAF2C84e1154c8DdD3203880e5db965bfF09B60](https://etherscan.io/address/0xDCAF2C84e1154c8DdD3203880e5db965bfF09B60) over 5 months starting 2021-12-01 as per their most recent [budget proposal](https://mips.makerdao.com/mips/details/MIP40c3SP33).  
+
+A total of **5,121,875 DAI** will be streamed to the Deco Fixed Rate ([DECO-001](https://mips.makerdao.com/mips/details/MIP39c2SP23)) Core Unit at [0xF482D1031E5b172D42B2DAA1b6e5Cbf6519596f7](https://etherscan.io/address/0xF482D1031E5b172D42B2DAA1b6e5Cbf6519596f7) over 33 months starting 2021-12-01 as per their most recent [budget proposal](https://mips.makerdao.com/mips/details/MIP40c3SP35).  
```

Comment:
The MIP for this says to be streamed over 36 months, but the spreadsheet says 33, just want to check this is right?

Reply:
The mip also says 33 months (in the implementation section)

Reply:
3 months up-front, I think.

Reply:
I need to work on my reading comprehension